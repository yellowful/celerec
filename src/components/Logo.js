import React from 'react';

const Logo = () => {
    return(
        <div>
            <div id="logo" className="f2 tc ba bw1 br3 fl w-10 ml3 mt5 shadow-3 grow">
                logo brain
            </div>
        </div>
    )
}

export default Logo;