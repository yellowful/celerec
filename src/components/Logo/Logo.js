import React from 'react';
import Tilt from 'react-tilt'
import HeadLogo from './logo.png'


const Logo = () => {

    return (
        <Tilt className="Tilt" options={{ max: 25 }} style={{ height: 100, width: 100 }}>
            {/* 設定Tilt的大小 */}
            <header className="Tilt-inner w-100 tl ba bw1 br1 br3-ns mt1 shadow-2 grow">
                <img src={HeadLogo} alt="logo" />
            </header>
        </Tilt>
    )
}

export default Logo;