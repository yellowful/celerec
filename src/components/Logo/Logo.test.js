import React from 'react';
import { render } from '@testing-library/react'
import Logo from './Logo';

// logo應該要符合快照
describe('logo unit test', () => {
  it('should match snapshot', () => {
    const {asFragment} = render(<Logo />);
    expect(asFragment()).toMatchSnapshot();
  })
})