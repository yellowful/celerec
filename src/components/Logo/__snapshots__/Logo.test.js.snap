// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logo unit test should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Tilt"
    style="height: 100px; width: 100px;"
  >
    <header
      class="Tilt-inner w-100 tl ba bw1 br1 br3-ns mt1 shadow-2 grow"
    >
      <img
        alt="logo"
        src="logo.png"
      />
    </header>
  </div>
</DocumentFragment>
`;
